docs/includes/eloquent-models/PlanetDate.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace App\Models;
+
+use MongoDB\Laravel\Eloquent\Model;
+
+class Planet extends Model
+{
+    protected $casts = [
```

Comment:
This is going to change with Laravel 11. [Model Casts are moving to methods](https://laravel-news.com/model-casts)

Reply:
Thanks for the info, created a [ticket](https://jira.mongodb.org/browse/DOCSP-37112).